src/replica/mutation_log.h

```diff
@@ -345,6 +345,67 @@ class mutation_log : public ref_counter
     // get total size ithout lock.
     int64_t total_size_no_lock() const;
 
+    struct reserved_slog_info
+    {
+        size_t file_count;
+        int64_t log_size;
+        int smallest_file_index;
+        int largest_file_index;
+
+        std::string to_string() const
+        {
+            return fmt::format("reserved_slog_info = [file_count = {}, log_size = {}, "
+                               "smallest_file_index = {}, largest_file_index = {}]",
+                               file_count,
+                               log_size,
+                               smallest_file_index,
+                               largest_file_index);
+        }
+
+        friend std::ostream &operator<<(std::ostream &os, const reserved_slog_info &reserved_log)
+        {
+            return os << reserved_log.to_string();
+        }
+    };
```

Comment:
It's needed to add USER_DEFINED_STRUCTURE_FORMATTER for reserved_slog_info because we have upgraded libfmt.